.gitpod.yml

```diff
@@ -0,0 +1,30 @@
+github:
+  prebuilds:
+    master: true
+    branches: true
+    pullRequests: true
+    pullRequestsFromForks: true
+    addCheck: false
+    addComment: false
+    addBadge: true
```

Comment:
I feel like the check version would be less intrusive here, if it correctly links to the same point. Looking at the docs, the badge doesn't look *too bad* though.

Reply:
addCheck would make CI fail if pre-building for gitpod fails
addComment would add a comment to every PR
addBadge adds a button on every PR, which doesn't send notifications